### ISSUE TITLE
Warning: Failed child context type: Cannot read property 'bemBlock' of undefined

### ISSUE THREAD

**Issue by shyr1punk** [bem/create-bem-react-app#29]
### Description

Newly created `bem-react-app` has  in console `Warning: Failed child context type: Cannot read property 'bemBlock' of undefined`

```
Warning: Failed child context type: Cannot read property 'bemBlock' of undefined
    in App-Header (at App.js:12)
    in div
    in App (at index.js:7)
printWarning @ warning.js:33
warning @ warning.js:57
checkPropTypes @ checkPropTypes.js:52
processChildContext$1 @ react-dom.development.js:7854
invalidateContextProvider @ react-dom.development.js:7890
finishClassComponent @ react-dom.development.js:10262
updateClassComponent @ react-dom.development.js:10226
beginWork @ react-dom.development.js:10605
performUnitOfWork @ react-dom.development.js:12573
workLoop @ react-dom.development.js:12682
callCallback @ react-dom.development.js:1299
invokeGuardedCallbackDev @ react-dom.development.js:1338
invokeGuardedCallback @ react-dom.development.js:1195
performWork @ react-dom.development.js:12800
scheduleUpdateImpl @ react-dom.development.js:13185
scheduleUpdate @ react-dom.development.js:13124
scheduleTopLevelUpdate @ react-dom.development.js:13395
updateContainer @ react-dom.development.js:13425
(anonymous) @ react-dom.development.js:17105
unbatchedUpdates @ react-dom.development.js:13256
renderSubtreeIntoContainer @ react-dom.development.js:17104
render @ react-dom.development.js:17129
module.exports @ index.js:24
__webpack_require__ @ bootstrap 5dbe3d7…:657
fn @ bootstrap 5dbe3d7…:85
(anonymous) @ fetch.js:458
__webpack_require__ @ bootstrap 5dbe3d7…:657
module.exports.ctor.super_ @ bootstrap 5dbe3d7…:706
(anonymous) @ bundle.js:710
```
at file `webpack:///./~/prop-types/factoryWithTypeCheckers.js`
```
function checkType(isRequired, props, propName, componentName, location, propFullName, secret) {
     ...

      if (props[propName] == null) { <------ props is undefined
        if (isRequired) {
          if (props[propName] === null) {
            return new PropTypeError('The ' + location + ' `' + propFullName + '` is marked as required ' + ('in `' + componentName + '`, but its value is `null`.'));
          }
          return new PropTypeError('The ' + location + ' `' + propFullName + '` is marked as required in ' + ('`' + componentName + '`, but its value is `undefined`.'));
        }
        return null;
      } else {
        return validate(props, propName, componentName, location, propFullName);
      }
    }
```

### Environment

Run these commands in the project folder and fill in their results:

`node v8.4.0`
`npm v5.3.0`:

Then, specify:

1. Operating system: OSX 10.11.6
2. Browser and version: Chromium 57, Firefox 57.0b9, Safari 11

**Comment by awinogradov** [comment]
What `bem-react-core` version do you have installed?

**Comment by shyr1punk** [comment]
"version": "0.4.2"

**Comment by awinogradov** [comment]
https://github.com/bem/bem-react-core/issues/164

**Comment by Yeti-or** [comment]
@awinogradov is it fixed ?

**Comment by awinogradov** [comment]
Yes, but not released yet

**Comment by awinogradov** [comment]
Released in `bem-react-core@1.0.0-rc.6`
